pkg/agent/datapath/localBridge.go

```diff
@@ -275,6 +254,8 @@ func (l *LocalBridge) BridgeInitCNI() {
 		sw := l.OfSwitch
 		l.cniConntrackCommitTable, _ = sw.NewTable(CNI_CT_COMMIT_TABLE)
 		l.cniConntrackRedirectTable, _ = sw.NewTable(CNI_CT_REDIRECT_TABLE)
+		// cni mode doesn't open mac learning
```

Comment:
what about remove `arp-to-controller` flow

Reply:
add a ip learning switch